src/model.js

```diff
@@ -424,11 +434,24 @@ const BookshelfModel = ModelBase.extend({
     let columnNames, candidates;
     if (_.isArray(arguments[1])) {
       columnNames = arguments[1];
-      candidates = _.drop(arguments, 2);
     } else {
       columnNames = null;
-      candidates = _.drop(arguments);
     }
```

Comment:
Hey, but this changes do not change the original API, only change it if the morph values are specified (which wasn't part of the API before).

The call `this.morphTo('imageable', Site, Author)` continues to be valid (the registry plugin version also continues to be valid), the condition `isArray(arguments[1])` will give false, which effectively drops only one argument (`imageable`) and sets column names as `null`